README.md

```diff
@@ -108,3 +108,19 @@ for more information about submitting pull requests or changes to EarthPy.
 
 [BSD-3](https://github.com/earthlab/earthpy/blob/master/LICENSE)
 
+See [`earthpy/tests/conftest.py`](earthpy/tests/conftest.py) for fixture
```

Comment:
@nkorinek there is still text in the readme that should not be there. 

THe vignette create text could go here:

https://github.com/earthlab/earthpy/blob/master/CONTRIBUTING.rst 

but the make docs part doesn't belong in the readme. I suggest moving this to contributing.
the only update i should see to the readme in this PR is the LINK to this new vignette. this link going below "why earthpy" you will see a bulleted list of things earthpy can do. i'd like to create a link to each vignette there.

Reply:
@nkorinek  did you see this comment? i believe this should go in contributing or it may already be there...